source/exe/hot_restart_nop.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
```

Comment:
in an optimal world, this hot_restart_nop.h would also be used by the test code which does the same thing. You would need to rejigger slightly and maybe put this in source/common/hot_restart/hot_restart_nop.h. It's not a huge deal but if you feel like it.